### PR TITLE
Added Fissure spawner Dial input and tooltips

### PR DESCRIPTION
**Pull request by wofsauge**
image

### CODE REVIEW

**Comment by Zamiell** [comment]
can we change it to not use magic numbers?

File: BasementRenovator.py [hunk 1]
```diff
@@ -2050,19 +2050,42 @@ def customContextMenu(self, pos):
         v.valueChanged.connect(lambda x: self.changeProperty(1, x))
         menu.addAction(Speed)
 
-        # Distance/Angle
-        Distance = QWidgetAction(menu)
-        d = QSpinBox()
-        d.setRange(0, 15)
+        if self.entity.Type == 3001:
```
Comment (Zamiell) [inline BasementRenovator.py:2053]: here

Comment (wofsauge) [reply BasementRenovator.py]: fixed

**Comment by wofsauge** [comment]
new layout: 
image

**Comment by Zamiell** [comment]
well done

**Comment by Zamiell** [comment]
the magic numbers are slain

**Comment by Zamiell** [comment]
looks good to me

**Pull request merged**